src/autoconfig.ts

```diff
@@ -2,12 +2,11 @@ import { TemplateSettings } from './types'
 import autoconfigTemplate from './views/autoconfig.xml'
 import swig from 'swig-templates'
 
+// Mozilla Thunderbird
 export default (settings: TemplateSettings) =>
-  async (req: Request): Promise<Response> => {
+  async (): Promise<Response> => {
```

Comment:
Ah, this is due to an unused, `req`, correct?

Reply:
Yessir. Webstorm was yelling at me :)